<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>БД Курсовые работы</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Выполнить
операцию</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Таблица</string>
    </property>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>112</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Группа</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>112</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Студент</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>112</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Тема</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>112</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Источник</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>112</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Проект</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>112</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Связь</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Операция</string>
    </property>
    <widget class="QRadioButton" name="radioButton_11">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>141</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Прочитать все</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_12">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>112</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Прочитать</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_13">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>112</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Создать</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_14">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>112</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Изменить</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_15">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>112</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>331</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_5"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Дата создания</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_4"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_1"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Квалификация</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_3"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Факультет</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Дата создания</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_1">
       <property name="text">
        <string>Номер группы</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>501</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_6"/>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_7"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>812</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
